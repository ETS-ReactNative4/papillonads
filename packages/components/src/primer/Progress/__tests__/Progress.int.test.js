import { renderNormal, renderLarge, renderSmall } from './Progress.int.render'

describe('<Progress />', () => {
  describe('Render', () => {
    test('must match renderNormal()', () => {
      expect(global.renderToJSON(renderNormal())).toMatchSnapshot()
    })

    test('must match renderLarge()', () => {
      expect(global.renderToJSON(renderLarge())).toMatchSnapshot()
    })

    test('must match renderSmall()', () => {
      expect(global.renderToJSON(renderSmall())).toMatchSnapshot()
    })
  })
})
