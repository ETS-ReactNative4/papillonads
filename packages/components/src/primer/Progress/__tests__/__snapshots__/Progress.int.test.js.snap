// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Progress /> Render must match renderLarge() 1`] = `
<span
  className="Progress Progress--large"
>
  <span
    className="Progress-item bg-green"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</span>
`;

exports[`<Progress /> Render must match renderNormal() 1`] = `
<span
  className="Progress"
>
  <span
    className="Progress-item bg-green"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</span>
`;

exports[`<Progress /> Render must match renderSmall() 1`] = `
<span
  className="Progress Progress--small"
>
  <span
    className="Progress-item bg-green"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</span>
`;
